packages/atomic/src/index.html

```diff
@@ -171,7 +175,7 @@
       }
 
       :root {
-        --atomic-font-family: 'canada-type-gibson';
+        --atomic-font-family: 'canada-type-gibson', Lato, Arial, Helvetica, sans-serif;
```

Comment:
fallback is applied to the Options of the Select dropdown, who doesn't have gibson downloaded from the web since it's native.